### ISSUE TITLE
French translation

### ISSUE THREAD

**Issue by breversa** [mudlej/mj_pdf#32]
Hello and thanks for this app!

I’ve been using it since it was integrated within /e/OS (https://e.foundation/e-os/) but it’s currently lacking a french translation.

How can I help translate it?

**Comment by breversa** [comment]
Yes, I'm a native french speaker so I volunteer to translate (or at least start) that file. :-)

What is you favorite way to return it to you once translated?

**Comment by mudlej** [comment]
The standard method is to create a merge request on the project repository on Gitlab by adding a folder named `values-fr` and an XML file called `strings.xml`. In fact, just four days ago, someone did exactly this for [Italian](https://gitlab.com/mudlej_android/mj_pdf_reader/-/merge_requests/16).

But I'm fine with receiving it in any convenient way, and I can add it myself. You can attach it to a reply here, paste it directly, or use any other method that works for you.

For the format, you can refer to the existing examples for other languages, such as [Spanish](https://gitlab.com/mudlej_android/mj_pdf_reader/-/blob/main/app/src/main/res/values-es/strings.xml) or [Turkish](https://gitlab.com/mudlej_android/mj_pdf_reader/-/blob/main/app/src/main/res/values-tr/strings.xml).

**Comment by breversa** [comment]
Is that okay: https://gitlab.com/mudlej_android/mj_pdf_reader/-/merge_requests/18 ?

**Comment by matkes** [comment]
Hello. Is it OK for the polish translation?
[https://gitlab.com/mudlej_android/mj_pdf_reader/-/merge_requests/23](url)

**Comment by mudlej** [comment]
I merged it.
Thanks!
